WcaOnRails/app/views/competitions/_events_tab.html.erb

```diff
@@ -18,7 +18,14 @@
             <%= competition_event.event.name if round.number == 1 %>
           </td>
           <td><%= round.round_type.name %></td>
-          <td><%= round.format.name %></td>
+          <% round_cutoff = round.cutoff %>
```

Comment:
Any reason for making a variable? It's of the same length as `round.cutoff` anyway ^^

Reply:
Nop.